.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
         stages: [commit]
       - id: check-docstrings
         name: check-docstrings
-        entry: python3 ./stdlib/scripts/check-docstrings.py
```

Comment:
Why this change?

Reply:
I did not make this change, this appeared somehow after I updated the branch after https://github.com/modularml/mojo/commit/d51dad80d863fac2a268f9215e22b0a989ca9176